bundle/manifests/oadp.openshift.io_cloudstorages.yaml

```
@@ -99,9 +99,69 @@ spec:
+                        Producers of specific condition types may define expected values and meanings for this field,
+                        and whether the values are considered a guaranteed API.
+                        The value should be a CamelCase string.
+                        This field may not be empty.
+                      maxLength: 1024
+                      minLength: 1
+                      pattern: ^[A-Za-z]([A-Za-z0-9_,:]*[A-Za-z0-9_])?$
+                      type: string
+                    status:
+                      description: status of the condition, one of True, False, Unknown.
+                      enum:
+                      - "True"
```

YAMLlint (1.37.1)

[error] wrong indentation: expected 24 but found 22

(indentation)